### ISSUE TITLE
production file contains full paths from dev environment

### ISSUE THREAD

**Issue by whitecolor** [stealjs/steal-tools#198]
Well maybe this does not effect on the work but it is definetly INcorrect that built production files contain any paths from dev envinoment.

```
define("package.json!npm",["@ 
....
main:"rimraf.js",globalBrowser:{},browser:{}},{name:"readable-stream",version:"1.1.13",fileUrl:"file:d:\\MyCodeFolderPath\\my-app/node_modules/nodemailer/node_modules/hyperquest
```

Why it contains `file:d:\\MyCodeFolderPath` ? It should not be there.

**Comment by matthewp** [comment]
Yeah I agree, will take a look.

**Comment by whitecolor** [comment]
I think putting the whole package json is exessive either.

**Comment by olegstepura** [comment]
Relative paths may solve an issue. Maybe event better (in terms of used bytes) to just use a hash of path

**Comment by matthewp** [comment]
This will be fixed in 0.14.0
